--- gdm/gdm.conf.comomo	2006-02-23 15:13:14.000000000 +0900
+++ gdm/gdm.conf	2006-02-23 15:15:57.000000000 +0900
@@ -60,7 +60,7 @@
 Group=gdm
 # To try to kill all clients started at greeter time or in the Init script.
 # doesn't always work, only if those clients have a window of their own
-KillInitClients=true
+KillInitClients=false
 LogDir=/var/gdm
 # You should probably never change this value unless you have a weird setup
 PidFile=/var/run/gdm.pid
@@ -129,7 +129,7 @@
 # Distributions: Ship with this off.  It is never a safe thing to leave
 # out on the net.  Alternatively you can set up /etc/hosts.allow and
 # /etc/hosts.deny to only allow say local access.
-Enable=false
+Enable=true
 # Honour indirect queries, we run a chooser for these, and then redirect
 # the user to the chosen host.  Otherwise we just log the user in locally.
 HonorIndirect=true
@@ -272,7 +272,7 @@
 # and they will always be started.  Each line must start with a unique
 # number and that will be the display number of that server.  Usually just
 # the 0 server is used.
-0=Standard
+#0=Standard
 #1=Standard
 # Note the VTAllocation and FirstVT keys on linux.  Don't add any vt<number>
 # arguments if VTAllocation is on, and set FirstVT to be the first vt
