--- ./defaults.conf.orig	2006-06-11 23:10:30.000000000 +0900
+++ ./defaults.conf	2006-06-11 23:11:34.000000000 +0900
@@ -97,7 +97,7 @@
 
 # To try to kill all clients started at greeter time or in the Init script.
 # does not always work, only if those clients have a window of their own.
-#KillInitClients=true
+KillInitClients=false
 LogDir=/var/log/gdm
 # You should probably never change this value unless you have a weird setup.
 PidFile=/var/run/gdm.pid
@@ -206,7 +206,7 @@
 # and contains any other value or no value for CONSOLE.
 AllowRemoteRoot=false
 # This will allow remote timed login.
-AllowRemoteAutoLogin=false
+AllowRemoteAutoLogin=true
 # 0 is the most restrictive, 1 allows group write permissions, 2 allows all
 # write permissions.
 RelaxPermissions=0
@@ -253,7 +253,7 @@
 # access is another alternative but not the safest.  Firewalling port 177 is
 # the safest if you wish to have xdmcp on.  Read the manual for more notes on
 # the security of XDMCP.
-Enable=false
+Enable=true
 # Honor indirect queries, we run a chooser for these, and then redirect the
 # user to the chosen host.  Otherwise we just log the user in locally.
 #HonorIndirect=true
@@ -526,7 +526,7 @@
 # will always be started.  Each line must start with a unique number and that
 # will be the display number of that server.  Usually just the 0 server is
 # used.
-0=Standard
+#0=Standard
 #1=Standard
 # Note the VTAllocation and FirstVT keys on Linux and FreeBSD.  Don't add any
 # vt<number> arguments if VTAllocation is on, and set FirstVT to be the first
