--- ./defaults.conf.orig	2006-06-25 22:07:14.000000000 +0900
+++ ./defaults.conf	2006-06-25 22:06:07.000000000 +0900
@@ -97,7 +97,7 @@
 
 # To try to kill all clients started at greeter time or in the Init script.
 # does not always work, only if those clients have a window of their own.
-#KillInitClients=true
+KillInitClients=false
 LogDir=/var/log/gdm
 # You should probably never change this value unless you have a weird setup.
 PidFile=/var/run/gdm.pid
@@ -199,12 +199,12 @@
 [security]
 # Allow root to login.  It makes sense to turn this off for kiosk use, when
 # you want to minimize the possibility of break in.
-AllowRoot=false
+AllowRoot=true
 # Allow login as root via XDMCP.  This value will be overridden and set to
 # false if the /etc/default/login file exists and contains
 # "CONSOLE=/dev/login", and set to true if the /etc/default/login file exists
 # and contains any other value or no value for CONSOLE.
-AllowRemoteRoot=false
+AllowRemoteRoot=true
 # This will allow remote timed login.
 AllowRemoteAutoLogin=false
 # 0 is the most restrictive, 1 allows group write permissions, 2 allows all
@@ -253,7 +253,7 @@
 # access is another alternative but not the safest.  Firewalling port 177 is
 # the safest if you wish to have xdmcp on.  Read the manual for more notes on
 # the security of XDMCP.
-Enable=false
+Enable=true
 # Honor indirect queries, we run a chooser for these, and then redirect the
 # user to the chosen host.  Otherwise we just log the user in locally.
 #HonorIndirect=true
@@ -526,7 +526,7 @@
 # will always be started.  Each line must start with a unique number and that
 # will be the display number of that server.  Usually just the 0 server is
 # used.
-0=Standard
+#0=Standard
 #1=Standard
 # Note the VTAllocation and FirstVT keys on Linux and FreeBSD.  Don't add any
 # vt<number> arguments if VTAllocation is on, and set FirstVT to be the first
